pkg/remote/google/util.go

```diff
@@ -0,0 +1,23 @@
+package google
+
+import (
+	"fmt"
+	"strings"
+)
+
+type googleResourceAttributes struct {
+	ProjectName string
+	Region      string
+}
+
+func parseResourceName(name string) (googleResourceAttributes, error) {
+	attrs := googleResourceAttributes{}
+	splittedName := strings.Split(name, "/")
```

Comment:
As we talked during our live session the other day, we should use a regex here to be sure to extract things at the right index. We cannot predict that we'll always get the same structure for resource names.

For example for `google_compute_firewall` we do not have a region `//compute.googleapis.com/projects/cloudskiff-dev-elie/global/firewalls/test-firewall-0`

For project we should match something like `projects\/(.*?)\/`, same idea for region

Reply:
Since it's a non deep-mode implementation I'll undo that change and do it in another PR